### DISCUSSION TITLE
VS Code access to all LXCs?

### DISCUSSION THREAD

**Discussion by di5cord20** [tteck/Proxmox#3075] [Help Needed]
I'd like to use VS Code (and Filebrowser) to access all my LXCs.  I'm often ssh-ing into LXCs to modify a config.  VS Code could potentially make that approach easier.  I'm open to other suggestions as well.  I haven't figured out how to reload the config (Caddy) or do a docker compose refresh (Docker) without SSH though... is that possible through the terminal within VScode?

Typical use case:
- update Caddy Caddyfile and then reload Caddy (currently use ssh and nano to edit)
- modify docker compose on my Docker enabled LXCs followed by docker compose -up d command (current setup vscode as docker container and setup volume to grant access)

**Comment by di5cord20** [comment]
Just following up.  I'm curious how others may be configuring vscode (or Filebrowser) through Proxmox?  

- Are you installing on an LXC or Host?
- How do you access files/directories on other LXCs / VMs through VScode?
- Is this even possible?
- If it is possible, are you able to edit files within vscode/filebrowser? 

Edit: added another bullet

**Comment by Mellowlynx** [comment]
I don't use vscode to edit the configs.
I just SSH intro the Proxmox host and enter a LXC trough there. `pct list` and `pct enter ID` (ID is de LXC id)
And use nano or vim to edit the file I want to edit.

I'm not sure if what you want is possible, I think you have to add SSH to all your LXC's and connect directly, but I can be wrong.

**Reply by di5cord20** [reply]
I too am using ssh and nano but prefer vscode. When at home, I use vscode thick client on my desktop with the ssh remote extension but that doesn't work on the web version.

Perhaps what I'm looking for indeed is not possible?
